integration-test/2022-disputed-borders.py

```diff
@@ -74,7 +74,8 @@ def test_admin_level_3_state(self):
                 'boundary': 'administrative',
                 'place': 'state',
                 'source': 'openstreetmap.org',
-                'admin_level:XX': '2'
+                'admin_level:XX': '2',
+                'recognized_by:': 'YY'
```

Comment:
Please extend this test so there's a disputing point of view, too which would result in an `unrecognized_disputed_reference_line` in the test below.